src/main/scala/BintrayRepo.scala

```diff
@@ -48,6 +48,7 @@ case class BintrayRepo(credential: BintrayCredentials, org: Option[String], repo
         }
       if (!exists) sys.error(
         s"was not able to find or create a package for $owner in $repo named $packageName")
+      log.debug(s"Requested vcs URL: $vcs")
```

Comment:
This might help. However I think it would be more interesting to log the request and responses to/from bintray. But that is done by the bintry lib.